### PR TITLE
Update code to support multi-arch job

### PR DESCRIPTION
**Pull request by liqcui**
Task:
This PR used for enabling data-path-9nodes, control-plane-24nodes, density-heavy-24nodes job for the profile AWS-Multiarch-Graviton-Baremetal-OVN
Jira Ticket
https://issues.redhat.com/browse/PERFSCALE-2747

### CODE REVIEW

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-multi-control-plane-24nodes

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-multi-control-plane-24nodes

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-multi-control-plane-24nodes

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-multi-control-plane-24nodes

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-multi-control-plane-24nodes

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-multi-node-density-heavy-24nodes

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-multi-data-path-9nodes

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-multi-data-path-9nodes

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-multi-data-path-9nodes

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-multi-control-plane-24nodes

**Comment by liqcui** [comment]
/pj-rehearse  periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-multi-node-density-heavy-24nodes

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-multi-data-path-9nodes

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-multi-node-density-heavy-24nodes

**Comment by qiliRedHat** [comment]
@liqcui From https://docs.google.com/document/d/1NbXwfZwSkMdftuKW1Ob705rQk-XHLtPtP1d1vuqF9Vo, the AWS-OVN-Multiarch-Gravition-Baremetal is a Manual Triggered profile. We can add a job but should be always-run: false.

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-multi-data-path-3nodes

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.16-nightly-multi.yaml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    allow_skip_on_success: true
+    cluster_profile: aws-perfscale-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: aarch64
+      ADDITIONAL_WORKER_VM_TYPE: m7g.metal
+      ADDITIONAL_WORKERS: "2"
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "1"
```
Comment (qiliRedHat) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.16-nightly-multi.yaml:70]: I'm not sure if this config can get a successful install. I triggered a test job to confirm.

**Comment by liqcui** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-multi-data-path-4nodes

**Pull request merged**